### PR TITLE
Fixed issue of validation

### LINKED ISSUE #2547: When Terms and Condition checkbox is required Google pay and apple not validating it
Making your own contribution is easy, encouraged and greatly appreciated! We will put effort into reviewing and merging your PR quickly. For more info, please refer to the contribution guidelines: https://github.com/Adyen/adyen-magento2/blob/develop/CONTRIBUTING.md

**Describe the bug**
Activate terms and conditions check box and make it required field. and tru to place order with Google pay and apple pay, it will not validate the term and condition checkbox and order will be placed successfully without selecting it.

**To Reproduce**
1. Go to Admin panel and activate t&c as required field 
2. Click on on Google pay or apple pay button from the checkout page payment section
4.it will show Google payment form directly 

**Expected behavior**
It should validate the t&c checkbox before showing Google pay payment form.

**Magento version**
2.4.6.-p1

**Plugin version**
9.2.0

**Screenshots**

**Desktop (please complete the following information):**
 - OS: All
 - Version All

### PR DESCRIPTION
**Pull request by sarfarazbheda**
**Description**

When the terms and condition checkbox is required field it was not validating the Checkbox.

**Tested scenarios**

Set terms and condition as required field on the checkout page

Fixes #2547

### CODE REVIEW

**Comment by sarfarazbheda** [comment]
Please approve workflows

File: view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js
Comment (khushboo-singhvi) [inline view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js:1]: Hey @sarfarazbheda, Thank you for the PR. We appreciate your contribution to the plugin. 
After reviewing your changes, I noticed that the onClick event has been applied to all PaymentMethods. While your approach is commendable, we need to ensure that this event is only triggered for specific PaymentMethods, such as ApplePay, GooglePay, etc. 
I would kindly request that you refactor the PR to include it only for the necessary PaymentMethods or else we can handle the refactoring within our team's availability.

Comment (sarfarazbheda) [reply view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js:1]: I have checked with all payment methods and working fine. You can test with different methods to confirm.

**Comment by Axel29** [comment]
Any update on this issue / PR ?

It's been months now and the bug is still present...

Thank you.

**Comment by candemiralp** [comment]
Hello @sarfarazbheda,

Thank you for you contribution and we appreciate all the effort. Unfortunately, this PR can not be merged to due to the reasons explained [here](https://github.com/Adyen/adyen-magento2/issues/2547#issuecomment-2336813607).

Our team will provide a fix soon to this known bug.

Best Regards,
Can

**Pull request closed without merging**